features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/APIDefinition/APIDefinition.jsx

```diff
@@ -198,6 +164,14 @@ class APIDefinition extends React.Component {
         this.setState({ swagger: formattedString, format: convertTo, convertTo: format });
     }
 
+    setSchemaDefinition=(swagger, graphQL) => {
```

Comment:
There should be a Eslint issue here

Reply:
It doesn't see any Eslint issue in here

Reply:
Are we good? Shall we merge @dushansilva

Reply:
I checked locally for some reason that eslint rule is being ignore for this file only, not sure why, typically there should be a space before and after the = sign

Reply:
prod build was pass

Reply:
Prod build was passed